recipes/cppzmq/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "cppzmq" %}
+{% set version = "4.2.1" %}
+{% set sha256 = "11c699001659336c7d46779f714f3e9d15d63343cd2ae7c1905e4bf58907cef9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/zeromq/cppzmq/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win and py27]
```

Comment:
This is `win and py27` not `win or py27`, which means other combinations of `win`+`pyXX` should work.
So this recipes is missing a build dependency on Python, to activate the `vc`, and the proper `vc`s features.